ironfish-cli/src/commands/wallet/chainport/send.ts

```diff
@@ -0,0 +1,346 @@
+import { getExplorer } from '../../../utils/explorer'
+import { selectFee } from '../../../utils/fees'
+import { watchTransaction } from '../../../utils/transaction'
+
+/**
+ * {
+    "bridge_output": {
+        "publicAddress": "3be494deb669ff8d943463bb6042eabcf0c5346cf444d569e07204487716cb85",
+        "amount": "100",
+        "memoHex": "000234800b2c018338d0890c72c738404330c3cc38b2872c304a14634c2c72c6",
+        "assetId": "51f33a2f14f92735e562dc658a5639279ddca3d5079a6d1242b2a588a9cbf44c"
+    },
+    "gas_fee_output": {
+        "publicAddress": "93f6411753fd48062d02278fb55d9a5d53307794b06d31f4026821c93d5dca48",
+        "amount": "3",
+        "memo": "{\"type\": \"fee_payment\"}"
+    },
+    "bridge_fee": {
+        "source_token_fee_amount": 1,
+        "portx_fee_amount": 0,
+        "is_portx_fee_payment": false
+    }
+}
+ */
+
+type BridgeOutput = {
+  publicAddress: string
+  amount: string
+  memoHex: string
+  assetId: string
+}
+
+type GasFeeOutput = {
+  publicAddress: string
+  amount: string
+  memo: string
+}
+
+type BridgeFee = {
+  source_token_fee_amount: number
+  portx_fee_amount: number
+  is_portx_fee_payment: boolean
+}
+
+type BridgeTransactionDetails = {
+  bridge_output: BridgeOutput
+  gas_fee_output: GasFeeOutput
+  bridge_fee: BridgeFee
+}
+
+export class BridgeCommand extends IronfishCommand {
+  static description = `Get fee distribution for most recent blocks`
+  static hidden = true
+
+  static flags = {
+    ...RemoteFlags,
+    watch: Flags.boolean({
+      default: false,
+      description: 'Wait for the transaction to be confirmed',
+    }),
+    account: Flags.string({
+      char: 'f',
+      description: 'The account to send money from',
+    }),
+    to: Flags.string({
+      char: 't',
+      description: 'The public address of the recipient',
+    }),
+    amount: ValueFlag({
+      char: 'a',
+      description: 'The amount to send in the major denomination',
+      flagName: 'amount',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const client = await this.sdk.connectRpc()
+    const { flags } = await this.parse(BridgeCommand)
+    let from = flags.account?.trim()
+    let to = flags.to?.trim()
+
+    const networkId = (await client.chain.getNetworkInfo()).content.networkId
+
+    if (networkId !== TESTNET.id) {
+      CliUx.ux.error('This command is only available on testnet')
+    }
+
+    if (!from) {
+      const response = await client.wallet.getDefaultAccount()
+
+      if (!response.content.account) {
+        this.error(
+          `No account is currently active.
+           Use ironfish wallet:create <name> to first create an account`,
+        )
+      }
+
+      from = response.content.account.name
+    }
+
+    if (!to) {
+      to = await CliUx.ux.prompt('Enter the public address of the recipient', {
+        required: true,
+      })
+    }
+
+    if (validator.isAddress(to) === false) {
+      this.error('Invalid to ethereum address')
```

Comment:
Not all supported chains are ethereum right? I looked it up and BSC is not standard ethereum addresses. Wondering about other chains too.

Also I think chain selection first makes sense intuitively, would also allow per chain validation of addresses.